### PR TITLE
[ROSAUTOTEST] Output '[ROSAUTOTEST] ' "everywhere"

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Follow-up to #1531.

Cc @ColinFinck

## Proposed changes

- Output '[ROSAUTOTEST]' "everywhere", except test logs.
Tell me if I missed other ways to output.
- A few minor fixes.

## TODO

- [X] Create TestMan matching PR. // Done: web PR9.
(SysReg2 should be fine as is.)
- [ ] Try them together. (I can't do that.)

### CODE REVIEW

File: modules/rostests/rosautotest/CJournaledTestList.cpp [hunk 1]
```diff
@@ -154,7 +154,7 @@ CJournaledTestList::WriteInitialJournalFile()
     CTestInfo* TestInfo;
     DWORD BytesWritten;
 
-    StringOut("Writing initial journal file...\n\n");
+    StringOut("[ROSAUTOTEST] Writing initial journal file...\n\n");
```
Comment (tkreuzer) [inline modules/rostests/rosautotest/CJournaledTestList.cpp]: Can't we just have this done in StringOut?

Comment (SergeGautherie) [reply modules/rostests/rosautotest/CJournaledTestList.cpp]: Not as is, see comments on #1531: that would probably need to create a `StringOutRAT()` or `StringOutTest()`...

**Comment by SergeGautherie** [comment]
Would it help if I "postpone" the few changes that affect TestMan too?

**Comment by ColinFinck** [comment]
This is not worth the hassle

**Pull request closed without merging**